lib/utils/paginator.js

```diff
@@ -8,16 +8,25 @@ var chalk = require('chalk');
  * a subset of the choices if the list is too long.
  */
 
-var Paginator = module.exports = function () {
+var Paginator = module.exports = function (screen) {
   this.pointer = 0;
   this.lastIndex = 0;
+  this.screen = screen;
 };
 
 Paginator.prototype.paginate = function (output, active, pageSize) {
   pageSize = pageSize || 7;
   var middleOfList = Math.floor(pageSize / 2);
   var lines = output.split('\n');
 
+  if (this.screen) {
+    lines = this.screen.breakLines(lines);
```

Comment:
Hey, I feel like it'd be pretty simple to make `breakLines` and `normalizedCliWidth` into fully standalone functions.

Then we wouldn't need to keep track of whether or not we received a ScreenManager instance.

Reply:
Thanks for the feedback. I'm happy to make whatever changes will help maintenance.

Since `normalizedCliWidth` needs an `output` value, I assume that that would be passed in as a parameter to `breakLines` and `normalizedCliWidth`?

Where would you like the standalone functions to reside? `utils.js`? Static functions on `ScreenManager`?

Reply:
They could be static on `ScreenManager` for now at least.

Reply:
The readline instance (e.g. used in `this.rl.output`) is always available on the UI classes as `this.rl`. So yeah, we can pass it in as an argument.